server/routes/bookmark.js

```diff
@@ -14,4 +15,11 @@ router.post('/articles/:slug/bookmarks',
   checkIfArticleExists,
   tryCatchWrapper(Bookmark.bookmarkArticle));
 
+router.delete('/articles/:slug/bookmarks/:id',
```

Comment:
This url doesn't seem correct. Isn't a bookmark related with a user? This url should be /user/:id/bookmarks

Reply:
@seyi-adeleke The route is for un-bookmarking an article. 

I believe that bookmarks are associated with the article being bookmarked. It is just like comments, complaints and reactions (Like and Love). And since every bookmarking happens on the article via `POST /articles/:slug/bookmarks` route, I followed similar logic in implementing the **un-bookmarking**. 

I think the route you suggested will be most suitable for:
> getting all articles bookmarked by a user. (all bookmarks by a user) `/users/:username/bookmarks`